editoast/src/models/timetable.rs

```diff
@@ -56,6 +59,29 @@ impl Timetable {
             .await
             .map_err(Into::into)
     }
+
+    pub async fn schedules_in_time_window(
```

Comment:
```suggestion
    pub async fn schedules_before_date(
```

since we're going this way